OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -66,118 +71,127 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <msItem xml:id="ms_i1.2.1">
                                     <locus from="107r" to="108r"/>
                                     <title type="complete" ref="LIT1828Mahale#Moses1"/>
-                                    <note/>
                                     <incipit xml:lang="gez">
-                                        <lb/>  <hi rend="rubric">ጸሎተ፡ ሙሴ፡ ነቢይ፡ ዘቀዳሚት፡ ሕግ።
-                                        <lb/>  ፩። ንሴብሕ፡ ለእግዚአብሔር፡ ስቡሕ፡ ዘተሰብሐ።</hi>
-                                        <lb/> ፈረሰ፡ ወመስተፅዕነ፡ ወረወ፡ ውስተ፡ ባሕር።
+                                        <locus target="#107r"/>
+                                        <lb/><hi rend="rubric">ጸሎተ፡ ሙሴ፡ ነቢይ፡ ዘቀዳሚት፡ ሕግ።
+                                            <lb/>፩። ንሴብሕ፡ ለእግዚአብሔር፡ ስቡሕ፡ ዘተሰብሐ።</hi>
+                                        <lb/>ፈረሰ፡ ወመስተፅዕነ፡ ወረወ፡ ውስተ፡ ባሕር።
                                     </incipit>
                                     <explicit xml:lang="gez">
-                                        <lb/>  ወአስተጋብአ፡ <hi rend="rubric">እግዚአብሔር፡</hi> ላዕሌሆሙ፡ ማየ፡ ባሕር።
-                                        <lb/>  ወደቂቀ፡ እስራኤልሰ፡ ኃለፉ፡ እንተ፡ የብስ፡ በማዕከለ፡ <add place="above">ባሕር።</add> 
-                                        <lb/> ማይሰ፡ ሎመ፡ አረፍት፡ በይምን፡ ወአረፍት፡ በጽግም።
+                                        <locus target="#108r"/>
+                                        <lb/>ወአስተጋብአ፡ <hi rend="rubric">እግዚአብሔር፡</hi> ላዕሌሆሙ፡ ማየ፡ ባሕር።
+                                        <lb/>ወደቂቀ፡ እስራኤልሰ፡ ኃለፉ፡ እንተ፡ የብስ፡ በማዕከለ፡ <add place="above">ባሕር።</add> 
```

Comment:
this is actually `<seg rend="above">`, because it is not a later addition (here and later)